checks/file_utils.go

```diff
@@ -106,3 +130,21 @@ func CheckFilesContent(shellPathFnPattern string,
 
 	return res, nil
 }
+
+// ContainsCommands checks if the file content contains commands or not.
+// `comment` is the string or character that indicates a comment.
```

Comment:
change |comment| to |command| in this function and comments.

Reply:
it's confusing. The function is used to see if the function contains commands. But it does so by checking if it contains only comments or empty lines. If we use comments instead of commands, the function should be called `ContainsOnlyCommentsOrEmptyLines` and I should change the return values false <-> true

Reply:
changed to `CheckFileContainsCommands`. Can update in follow-up PR if necessary.